src/adapters/places.ts

```diff
@@ -34,8 +34,25 @@ export async function createPlacesComponent(
     return response.data[0]
   }
 
+  async function getPlaceStateById(ids: string[]): Promise<Pick<PlaceAttributes, 'id' | 'disabled'>[]> {
+    const response = await fetch.fetch(`${placesApiUrl}/places/status`, {
```

Comment:
If there are more than the limit of places that can be queried at the same time on the ids list, the request will fail. We need to either repeat the request here or in the method that uses this one, by doing it once for each block of ids until.